tests/test_writer.py

```diff
@@ -169,6 +169,56 @@ def test_vcd_scopes(capsys):
         assert line.startswith(expected)
 
 
+def test_vcd_scopes_timestamp(capsys):
+    with VCDWriter(sys.stdout, date='today', init_timestamp=123) as vcd:
+        vcd.set_scope_type('eee.fff.ggg', 'task')
+        vcd.register_var('aaa.bbb', 'nn0', 'integer', 8, init='z')
+        vcd.register_var('aaa.bbb', 'nn1', 'integer', 8)
+        vcd.register_var('aaa', 'nn2', 'integer', 8)
+        vcd.register_var('aaa.bbb.ccc', 'nn3', 'integer', 8)
+        vcd.register_var('aaa.bbb.ddd', 'nn4', 'integer', 8)
+        vcd.register_var('eee.fff.ggg', 'nn5', 'integer', 8)
+        vcd.set_scope_type('aaa.bbb', 'fork')
+        with pytest.raises(TypeError):
+            vcd.set_scope_type({'a', 'b', 'c'}, 'module')
+
```

Comment:
I'd like to see this test scaled-down to just test the new `init_timestamp` parameter. To test that effectively, we only need one var. The test name should probably be `test_init_timestamp()`.